common/changes/@uifabric/icons/icon-page-pivot_2019-06-24-20-48.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/icons",
+      "comment": "Update MDL2 icons. Add AllIconNames.json",
+      "type": "patch"
```

Comment:
This is not a patch. You are adding icons. Adding functionality means "minor". :)

Reply:
Whoops, thanks for catching that.